### PR TITLE
feat(appeals): add submission id

### PR DESCRIPTION
**Pull request by bryn-pins**
adds an appeals case submission id to command and data model

### CODE REVIEW

File: schemas/appeal-has.schema.json [hunk 1]
```diff
@@ -90,6 +90,12 @@
       "examples": ["7002471"],
       "description": "External case identifier"
     },
+    "submissionId": {
+      "type": ["string", "null"],
```
Comment (bryn-pins) [inline schemas/appeal-has.schema.json:94]: nullable as migrated appeals won't have a submission id

File: schemas/commands/appellant-submission.schema.json [hunk 2]
```diff
@@ -227,6 +233,7 @@
         }
       },
       "required": [
+        "submissionId",
```
Comment (bryn-pins) [inline schemas/commands/appellant-submission.schema.json:236]: required as we're expecting to always send from FO

**Review by fabfab-kc — CHANGES_REQUESTED** [review]
Looks good to me. I have a doubt if the new field should appear in the required properties, but is correct as it is - being an update to an existing schema...

We also need to up the package.json version (and tag the repo when we merge)

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pins-data-model",
-  "version": "1.7.2",
+  "version": "1.7.3",
```
Comment (bryn-pins) [inline package.json:3]: @fabfab-kc 
I think I did it already this morning, did you see an old copy, or is there more to do?

Comment (fabfab-kc) [reply package.json:3]: sorry must have missed it!

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**